src/components/aktivitetskrav/vurdering/AktivitetskravVurderingAlert.tsx

```diff
@@ -82,6 +82,7 @@ export const AktivitetskravVurderingAlert = ({
       );
     }
     case AktivitetskravStatus.STANS:
+    case AktivitetskravStatus.FORHANDSVARSEL:
```

Comment:
Det bør vel implementeres en eller annen form for alert også når siste vurdering er Forhåndsvarsel

Reply:
Godt poeng, det tror jeg vi ikke har tenkt på 🤔